### PR TITLE
US96 Filter Form v2

### PR DESCRIPTION
**Pull request by aneuhold**
### Overview
Implements `v-model` with `FilterForm` and finishes up some other state conversions that were pending. 

### Includes:
- client/src/components/FilterForm.vue 
- client/src/components/InputField.vue
- client/src/views/Create.vue 

### Developer Checklist:
- [x] The code follows the Quality Policy file on Google Drive
- N/A My code has been developer-tested and includes unit tests
- [x] I have considered proper use of exceptions
- [x] I have eliminated IDE warnings

### Refs:
- [US 96](https://the-halo-mod.atlassian.net/jira/software/projects/HM/boards/1?selectedIssue=HM-96)

### CODE REVIEW

File: client/src/components/FilterForm.vue [hunk 1]
```diff
@@ -53,38 +49,57 @@ export default {
...
+    DoubleField,
   },
   methods: {
-    createSetCurrentValueFunc(filterType, varName) {
+    createSetParamValueFunc(filterType, varName) {
       return (newValue) => {
         // Set the new value temporarily
-        this.filterParams[filterType][varName] = newValue;
+        this.filterData.filter_params[varName] = newValue;
+        this.allFilterData[filterType][varName] = newValue;
```
Comment (Tengelma) [inline client/src/components/FilterForm.vue:77]: Same feedback as before. Don't write to props!

**Review by Tengelma — CHANGES_REQUESTED** [review]
Just the one change -- no writing to props!

File: client/src/components/FilterForm.vue (hunk 1 above)
Comment (aneuhold) [reply client/src/components/FilterForm.vue:77]: Same comment as in the other PR then looking for feedback on how to do this instead.

Comment (aneuhold) [reply client/src/components/FilterForm.vue]: Done

**Review by Tengelma — APPROVED** [review]
Looks good! thanks for making the suggested changes.

**Pull request merged**